google/ads/google_ads/util.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# TODO - Remove patching ToJsonString once issue
+# has been addressed in the protobuf library.
+def patch_to_json_method(obj):
+    """Patches an object's ToJsonString method.
+
+    Args:
+        obj: An arbitrary object.
+    """
+    obj.ToJsonString = _patched_to_json_string
+
+
+def _patched_to_json_string(self):
+    """Converts all string in an objects paths array to camel case."""
+    try:
+        iter(self.paths)
+    except TypeError:
+        paths = []
+    else:
+        paths = self.paths
+
+    camelcase_paths = []
+
+    for path in paths:
+        camelcase_paths.append(_snake_case_to_camel_case(path))
+
+    return ','.join(camelcase_paths)
+
+
+def _snake_case_to_camel_case(path_name):
+  """Converts a path name from snake_case to camelCase.
+
+  This is a slightly modified version of the function defined in the protobuf
```

Comment:
Would be useful to mention what modifications were made. Who knows when the original will be fixed.